### DISCUSSION TITLE
Password reset e-mail issues

### DISCUSSION THREAD

**Discussion by vb-03** [pretix/pretix#2938] [Q&A] [answered]
Hi,
I've run into an issue with a Docker deployment where e-mails are sent put for everything... except for password resets.
I'm using Gmail's SMTP, and orders related e-mails are sent out immediately.
However, when it's a password reset, user requested or requested by an admin, according to the logs, it creates the task but it keeps retrying until it gives up.
Where should I look?

**Comment by vb-03** [comment]
![image](https://user-images.githubusercontent.com/47537857/205120202-eca9dfa8-f7df-476c-b30b-bab0a43f5ea6.png)
![image](https://user-images.githubusercontent.com/47537857/205120252-691a0ba7-3124-4d10-a7b5-5b04931fd1c8.png)

**Comment by raphaelm** [comment, marked as answer]
Is the GMail smtp set through the pretix interface or through pretix.cfg?

**Reply by vb-03** [reply]
Through the web interface

**Reply by raphaelm** [reply]
The web interface only allows setting smtp credentials for a specific organizer. Password resets are not related to an Organizer, so those settings can't be used, the global email settings from pretix.cfg are used instead.

**Reply by vb-03** [reply]
Ohh, that makes sense! Changed the config and it's working!
Thank you for the help and for the amazing work on this app!
